client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -179,7 +182,7 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 export default connect(
 	state => ( {
 		contactDetailsExtra: getContactDetailsExtraCache( state ),
-		userWpcomLang: getCurrentUserLocale( state )
+		userWpcomLangIsFr: getCurrentUserLocale( state ).match( /^fr-?/i )
```

Comment:
If the user is not in the state, the null here will error. I suspect a catastrophic one :)

That's not a problem for checkout (you can't get this deep without having a valid user) but we'll need to look into it more closely if we want to use it for nux.